### DISCUSSION TITLE
Public IP tor network

### DISCUSSION THREAD

**Discussion by 0x14Rp** [fastfetch-cli/fastfetch#1147] [General]
Is it possible to use the publicip module with Tor? I mean something like running torsocks fastfetch when my Tor service is on. I tried to use a custom module and execute command, but it didn't work
<img width="435" alt="image" src="https://github.com/user-attachments/assets/c8f11523-624f-4a84-abb1-c06bb8356b13">

**Comment by CarterLi** [comment]
You should use command module

**Reply by 0x14Rp** [reply]
I tried but for some reason it doesn't get the ip under tor , it always shows my ip, even I used the commands but they don't execute.
